### ISSUE TITLE
useless

### ISSUE THREAD

**Issue by outgaze** [heashey/Skyrant-Triggerbot-Source#1]
your cracked version is returning an error and won't run.

**Comment by outgaze** [comment]
> if you are talking about the .exe, make sure to have siticone.dll and skyrant.exe in the same folder. if you are talking about source, you will have to fix the errors before u build it

i did it still errors, ill get a image give me a bit.

**Comment by outgaze** [comment]
![explorer_2tIZyyWV5a](https://user-images.githubusercontent.com/44008063/226422579-fa700c10-6150-4b60-a7b8-8f8c4f7291a4.png)

**Comment by heashey** [comment]
did not see this, please try now.

**Comment by reezliabi** [comment]
exe works fine but why crack p2c and release it on github/uc lmao

**Comment by outgaze** [comment]
> exe works fine but why crack p2c and release it on github/uc lmao

Yeah on win 10 but not on win 11 it gives the error as shown on the ss

**Comment by meowsynth** [comment]
> exe works fine but why crack p2c and release it on github/uc lmao

p2c?

**Comment by outgaze** [comment]
> > exe works fine but why crack p2c and release it on github/uc lmao
> 
> p2c?

paste 2 cheat XD? idk what he means but this is def not a p2c also it will probably get detected fast with time unless full source is released to modify it or you change the hell out of the signatures and properties

**Comment by meowsynth** [comment]
> > > exe works fine but why crack p2c and release it on github/uc lmao
> > 
> > 
> > p2c?
> 
> paste 2 cheat XD? idk what he means but this is def not a p2c also it will probably get detected fast with time unless full source is released to modify it or you change the hell out of the signatures and properties

detect pixel trigger?

**Comment by heashey** [comment]
> > > exe works fine but why crack p2c and release it on github/uc lmao
> > 
> > 
> > p2c?
> 
> paste 2 cheat XD? idk what he means but this is def not a p2c also it will probably get detected fast with time unless full source is released to modify it or you change the hell out of the signatures and properties

this is skyrant, which is a p2c. This is also the full source, just because there is errors if you try to compile doesn't mean it isn't. The fixes are easy, pretty much just renaming stuff. You also don't need to change the signatures and properties KEKW, if you are that paranoid just obfuscate it and that'll do the job

**Comment by outgaze** [comment]
> > 
> 
> I released a version that is usable on uc, meaning I removed everything that says skyrant and made it free to the public to use, just like a general uc release. The source on github, will however include skyrant in the source files, but the download on github and uc will not.

It would be interesting to see their v2 cracked as well as supposedly has win 11 support although you claim their v2 is no different to the v1 but the v1 doesn't work on win 11 also isn't their vipercheats also a overpriced trigger bot? From what I've seen of the videos on YouTube

**Comment by meowsynth** [comment]
v1 work on w11 well, but v2 better ofc, be happy if it was hacked
